.eslintrc

```diff
@@ -7,6 +7,7 @@
   ],
 
   "env": {
+    "jest": true, 
```

Comment:
@gagoar Can you please separate Jest migration into separate PR?

Reply:
yea, this is a stacked PR. so It has files that shouldn't be there. fixing